include/ratelimit_manager.h

```diff
@@ -151,14 +152,25 @@ class RateLimitManager
         // Get all rules as json
         const nlohmann::json get_all_rules_json();
 
+        // Get all throttled entities as json
+        const nlohmann::json get_all_throttled_entities_json();
+
+        // Delete throttled entity
+        const Option<nlohmann::json> delete_throttle_by_id(const uint64_t id);
+
         // Clear all rules
         void clear_all();
 
         // Internal function to set base time
         void _set_base_timestamp(const time_t& base_time);
 
+        // Init
+        Option<bool> init();
+
         // Set store
-        Option<bool> init(Store* store);
+        inline void set_store(Store* store) {
```

Comment:
Setters like this have to be avoided. For singletons, better to use `init()` convention.